src/main/windows/mainWindow.ts

```diff
@@ -65,6 +65,16 @@ function createMainWindow(options: {linuxAppIcon: string}) {
     const {maximized: windowIsMaximized} = savedWindowState;
 
     const spellcheck = (typeof Config.useSpellChecker === 'undefined' ? true : Config.useSpellChecker);
+    const isFullScreen = (savedWindowStateFullscreen: boolean | undefined) => {
+        if (global.args.fullscreen !== undefined) {
+            return global.args.fullscreen;
+        }
+
+        if (Config.startInFullscreen !== undefined) {
+            return Config.startInFullscreen;
+        }
+        return savedWindowStateFullscreen ?? false;
```

Comment:
Since the defaulted value of `Config.startInFullscreen` is always `false`, this will always return `false` since it will never be `undefined`. We should probably just check for `Config.startInFullscreen` and if it's falsy then we can default to the savedWindowState.